compiler/luci/service/src/Nodes/CirclePad.cpp

```diff
@@ -24,4 +28,10 @@ luci::CircleNode *CloneNodeLet<CN::OPQR>::visit(const luci::CirclePad *)
   return _graph->nodes()->create<luci::CirclePad>();
 }
 
+loco::TensorShape sinf::Algorithm::visit(const luci::CirclePad *node)
+{
+  auto paddings = loco::must_cast<luci::CircleConst *>(node->paddings());
+  return use_paddings(node, paddings);
+}
```

Comment:
```suggestion
namespace sinf
{

loco::TensorShape Algorithm::visit(const luci::CirclePad *node)
{
  auto paddings = loco::must_cast<luci::CircleConst *>(node->paddings());
  return use_paddings(node, paddings);
}

} // namespace sinf
```

Reply:
Why did you put this function in the sinf namespace?
It is not included in other codes. https://github.com/Samsung/ONE/blob/f963350244b946c36ae04cbca031df1371c1b6c8/compiler/luci/service/src/Nodes/CircleSoftmax.cpp#L33-L38
 Is there a special reason?

Reply:
`sinf` of `sinf::Algorithm::visit` is `sinf` namespace but doesn't look like it's in the namespace.
it would be better to explicitly express to be clear.

if other codes are also using `sinf::---` then would be better to revise them too.

Reply:
> `sinf` of `sinf::Algorithm::visit` is `sinf` namespace but doesn't look like it's in the namespace. it would be better to explicitly express to be clear.

I see. It would be more clear to explicitly express namespace.